src/components/FileDropper/FileDropper.component.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, FC, useRef, useState, ReactNode, ReactElement, Children, cloneElement } from 'react';
+import styles from './FileDropper.module.scss';
+
+interface DropContainerProps {
+  children?: ReactNode | ReactNode[];
+  placeholder?: string;
+}
+
+/**
+ * This component is only for using in Storybook for showcasing drag'n'drop functionality.
+ */
+export const FileDropper: FC<DropContainerProps> = ({ children, placeholder = `Drag and Drop a .glb file here` }) => {
+  const ref = useRef<HTMLDivElement>(null);
+  const [modelSrc, setModelSrc] = useState('');
+
+  const getBase64 = (file: File): Promise<string> =>
```

Comment:
As I said in direct, it doesn't seem to only handle base64 file data, in most cases it will be a standard glb blob without encoding. So this naming is confusing. I'd name it `getBinary` instead.

Reply:
The resolved `reader.result` is a base64 string of the provided file.
I think it's not necessary to change this as it's only for using within Storybook for demoing file upload in a sandbox.
Visage consumers can write their own handlers when they need to implement file uploading ;)